Makefile

```diff
@@ -72,32 +73,32 @@ statusgo-library: ##@cross-compile Build status-go as static library for current
 
 docker-image: ##@docker Build docker image (use DOCKER_IMAGE_NAME to set the image name)
 	@echo "Building docker image..."
-	docker build --build-arg "build_tags=$(BUILD_TAGS)" . -t $(DOCKER_IMAGE_NAME)
+	docker build --file $(DOCKER_FILE_PATH) --build-arg "build_tags=$(BUILD_TAGS)" . -t $(DOCKER_IMAGE_NAME)
 
 xgo-docker-images: ##@docker Build xgo docker images
 	@echo "Building xgo docker images..."
-	docker build xgo/base -t $(XGOIMAGE)
-	docker build xgo/ios-simulator -t $(XGOIMAGEIOSSIM)
+	docker build --file $(DOCKER_FILE_PATH) _assets/build/xgo/base -t $(XGOIMAGE)
```

Comment:
`--file` here is not needed. It should build a docker image based on `_assets/build/xgo/base/Dockerfile`.